specification/compute/resource-manager/Microsoft.Compute/stable/2017-12-01/compute.json

```diff
@@ -6314,6 +6314,15 @@
         "singlePlacementGroup": {
           "type": "boolean",
           "description": "When true this limits the scale set to a single placement group, of max size 100 virtual machines."
+        },
+        "zoneBalance": {
+          "type": "boolean",
+          "description": "Whether to force stictly even Virtual Machine distribution cross x-zones in case there is zone outage."
```

Comment:
is x-zones some domain specific name?

Reply:
Somehow. "x-zones" or "zones redundant" or "cross zones"  VMScaleSet.